### PR TITLE
Add test cases for verification of data integrity in the 'Custom' project

### PR DESCRIPTION
**Pull request by suarezgpablo**
I have added 6 test cases to verify data integrity in all the case studies from the 'Custom' project. Some bugs have been encountered while adding the test cases and have been fixed as well.

### CODE REVIEW

File: modevo-consistency/src/main/java/giis/modevo/consistency/Oracle.java [hunk 1]
```diff
@@ -85,7 +79,8 @@ private BoundStatementBuilder processResultSetRow(List<String> columnNames, Map<
 					Class<?> clase = stringValue.getClass();
 					classes.add(clase);
 					toBind.add(stringValue);
-				} else {
+				}
+				else {
```
Comment (javiertuya) [inline modevo-consistency/src/main/java/giis/modevo/consistency/Oracle.java:83]: fix formatting

File: modevo-consistency/src/main/java/giis/modevo/consistency/Oracle.java [hunk 2]
```diff
@@ -98,6 +93,25 @@ private BoundStatementBuilder processResultSetRow(List<String> columnNames, Map<
...
 	}
 
+	/**
+	 * Depending on the value obtained from the query, it adds to the numeric map or the string map
+	 */
+	private void addValueIntString(Map<String, Integer> intValues, Map<String, String> stringValues,
+			Map<String, String> columnTypes, String columnName, ResultSet rs) throws SQLException {
+		if (columnTypes.get(columnName).equalsIgnoreCase("INT")) {
+			int j = rs.getInt(columnName);
+			intValues.put(columnName, j);
```
Comment (javiertuya) [inline modevo-consistency/src/main/java/giis/modevo/consistency/Oracle.java:103]: simplify to a singe statement

File: modevo-consistency/src/main/java/giis/modevo/consistency/Oracle.java [hunk 3]
```diff
@@ -98,6 +93,25 @@ private BoundStatementBuilder processResultSetRow(List<String> columnNames, Map<
...
+		} else {
+			String obtainedString = rs.getString(columnName);
+			if (obtainedString == null) {
+				stringValues.put(columnName, "");
+			}
+			else {
+				stringValues.put(columnName, obtainedString);
```
Comment (javiertuya) [inline modevo-consistency/src/main/java/giis/modevo/consistency/Oracle.java:110]: this is a coalesce, simplify to 1 statement

File: modevo-consistency/src/test/java/test4giis/consistency/TestCheckConsistency.java [hunk 4]
```diff
@@ -55,7 +55,67 @@ public void testCustomV1NewColumn() throws IOException {
...
+		projectionBeforeEvo.put("table3", "SELECT book.id as idbook,author.id as idauthor,  book.publisher as publisher FROM author Inner join authorbook ON author.id = authorbook.idauthor Inner join book ON book.id = authorbook.idbook;");
+		projectionAfterEvo.put("table3", "SELECT book.id as idbook,author.id as idauthor,   book.publisher as publisher FROM author Inner join authorbook ON author.id = authorbook.idauthor Inner join book ON book.id = authorbook.idbook order by book.id DESC, author.id DESC;");
+		testConsistency(name.getMethodName(), projectionAfterEvo, "custom", projectionBeforeEvo);
+	}
+	@Test
+	public void testCustomV3TwoNewColumnsUsingNMRelationship() throws IOException {
+		Map<String, String> projectionAfterEvo = new HashMap<String, String>();
+		Map<String, String> projectionBeforeEvo = new HashMap<String, String>();
+		projectionBeforeEvo.put("table1", "SELECT author.id as idauthor, book.id as idbook, book.title as title FROM author Inner join authorbook ON author.id = authorbook.idauthor Inner join book ON book.id = authorbook.idbook;");
```
Comment (javiertuya) [inline modevo-consistency/src/test/java/test4giis/consistency/TestCheckConsistency.java:74]: unify SQL style criterion, e.g. inner join should be uppercase

File: modevo-script/src/main/java/giis/modevo/migration/script/execution/ScriptExecution.java [hunk 5]
```diff
@@ -43,57 +43,94 @@ public void execute(Script script, CassandraConnection c, String nameKeyspace) {
...
 		}	
 	}
+	/**
+	 * Creates a list of statements with its values replaced by the values included in the list.
+	 */
+	private List<String> replaceVariableNamesListValues(List<ColumnValue> listColumnValues,
+			String statementWithVariables) {
+		List<String> listStatements = new ArrayList<>();
+		String statementReplacedValues = statementWithVariables;
+		boolean replaced = false;
```
Comment (javiertuya) [inline modevo-script/src/main/java/giis/modevo/migration/script/execution/ScriptExecution.java:86]: this flag is not necessary, return from inside loop when found

**Review by javiertuya — CHANGES_REQUESTED** [review]

File: modevo-script/src/main/java/giis/modevo/migration/script/execution/ScriptExecution.java (hunk 5 above)
Comment (suarezgpablo) [reply modevo-script/src/main/java/giis/modevo/migration/script/execution/ScriptExecution.java]: Removed the flag, although with a different solution as there could be more than one replacement in the loop.

**Pull request merged**